src/main/java/net/atos/zac/util/IndexerenRESTService.java

```diff
@@ -33,36 +25,10 @@ public class IndexerenRESTService {
     @Inject
     private IndexeerService indexeerService;
 
-    @Inject
-    @ActiveSession
-    private Instance<HttpSession> httpSession;
-
     @GET
     @Path("herindexeren/{type}")
-    public HerindexerenInfo herindexeren(@PathParam("type") ZoekObjectType type) {
```

Comment:
Hmm, hang on, so in this PR you want to get rid of the entire way of 'marking for indexing' and then 'indexing'? I think that's way too big of a change and will have unintended consequences.